cmd/go-camo/main.go

```diff
@@ -81,6 +81,7 @@ type CLI struct {
 	MaxSize             int64         `name:"max-size" help:"Max allowed response size (KB)"`
 	ReqTimeout          time.Duration `name:"timeout" default:"4s" help:"Upstream request timeout"`
 	MaxRedirects        int           `name:"max-redirects" default:"3" help:"Maximum number of redirects to follow"`
+	MaxSizeRedirect     string        `long:"max-size-redirect" description:"URL to redirect when max-size is exceeded"`
```

Comment:
Is it correct that it starts with `long:`? Every other line starts with `name:` 🤔

Reply:
Good catch. It should be `name` indeed. I will fix in HEAD.